etcdserver/etcdserverpb/raft_internal.proto

```diff
@@ -26,6 +26,7 @@ message InternalRaftRequest {
   LeaseRevokeRequest lease_revoke = 9;
 
   AuthEnableRequest auth_enable = 10;
+  UserAddRequest user_add = 11;
```

Comment:
OK. Should I change the type name, too (including etcdserver/etcdserverpb/rpc.proto)?